internal/logger/logger.go

```diff
@@ -177,3 +178,21 @@ func (f *loggerFactory) writer(level string) io.Writer {
 		return os.Stdout
 	}
 }
+
+func ErrorLogs(filename string) (writer io.Writer, err error) {
```

Comment:
why are we doing seperate initialization of logs and not using the exisiting initialization?

Reply:
If we return io.writer form [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/logger/logger.go#L45) it may need major code changes that'w why prince suggested this way.